### PR TITLE
LPS-174238 Error while changing an Object's ERC when the object has a M:M relationship

### PR DESCRIPTION
**Pull request by diogomelo09**
[LPS-174238](https://liferay.atlassian.net/browse/LPS-174238)

### CODE REVIEW

**Comment by marcelabc** [comment]
Hey @cgmonte ,
I'm assigning this PR to you because it is frontend. Feel comfortable assigning to anybody from your squad :+1:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectManagementToolbar.tsx [hunk 1]
```diff
@@ -60,6 +60,9 @@ export default function ObjectManagementToolbar({
 			isRootDescendantNode={isRootDescendantNode}
 			label={label}
 			onExternalReferenceCodeChange={(externalReferenceCode: string) => {
+				if (!setValues) {
+					return;
+				}
```
Comment (cgmonte) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectManagementToolbar.tsx:65]: Hey @diogomelo09, thanks for working on this! I see that, with this PR, we are preventing `setValues` from being called and the save still works! How can that be? Either that `setValues` is useless or something that should be set is not being set. The real investigation here is to find out why `setValues` is undefined when the user navigates to tab other then "Details". And also evaluate if the implications of not executing this `setValues`.

**Review by cgmonte — CHANGES_REQUESTED** [review]
Hi, Diogo! Please read my comment as a I think that the suggested fix only hides the real problem. Also, the error displayed in the ticket's attached video is different from the one we are fixing here ("setValues is not a function"). Are you sure we a re dealing with the same problem here? The error in the video suggests a back-end bug: "Reverse object definition cannot be deleted".

**Comment by alinedoleron** [comment]
I'm closing this PR since this ticket was discarted.
https://liferay.atlassian.net/browse/LPS-174238

**Pull request closed without merging**